src/papi.c

```diff
@@ -3056,7 +3056,9 @@ PAPI_reset( int EventSet )
  *  The counters continue counting after the read. 
  *
  *  Note the differences between PAPI_read() and PAPI_accum(), specifically
- *  that PAPI_accum() resets the values array to zero.
+ *  that PAPI_accum() adds the current counter values to the existing counter
+ *  values held in the provided array (second parameter in PAPI_accum()) and resets the
```

Comment:
Note that the values stored in the array don't have to be "existing counter values" but can be any arbitrary values provided by the user.

I suggest changing the wording to the following:

"Note the differences between PAPI_read() and PAPI_accum(). Specifically, PAPI_accum() adds the counter values to the values stored in the array (the second parameter in PAPI_accum()) and then resets the counters to zero."

Reply:
I have gone ahead and updated both sections of text with your suggested changes!